pkg/virt-api/api.go

```diff
@@ -882,15 +882,14 @@ func (app *virtAPIApp) Run() {
 	// Run informers for webhooks usage
 	kubeInformerFactory := controller.NewKubeInformerFactory(app.virtCli.RestClient(), app.virtCli, app.aggregatorClient, app.namespace)
 
-	configMapInformer := kubeInformerFactory.ConfigMap()
+	kubeVirtCAConfigMapInformer := kubeInformerFactory.KubeVirtCAConfigMap()
 	// Wire up health check trigger
-	configMapInformer.SetWatchErrorHandler(func(r *cache.Reflector, err error) {
+	kubeVirtCAConfigMapInformer.SetWatchErrorHandler(func(r *cache.Reflector, err error) {
```

Comment:
It is slightly weird that we use this informer. I guess it is better than nothing ...